steps/terraform-version-nagger.yaml

```
@@ -17,6 +17,7 @@ steps:
     inputs:
       targetType: 'inline'
       script: |
+        pip3 install packaging
```

I would be explicit when calling pip. It's been causing some issues in the past when solely using pip3 shim. 
```suggestion
        python3 -m pip install packaging
```